INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 284-814-612
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 284-814-612
DEBUG views.py:8 我是debug级别日志
INFO views.py:9 我是infog级别日志
WARNING views.py:10 我是warning级别日志
ERROR views.py:11 我是error级别日志
CRITICAL views.py:12 我是critical级别日志
DEBUG views.py:14 falsk中记录的debug日志
INFO _internal.py:88 127.0.0.1 - - [12/Oct/2018 19:54:40] "GET / HTTP/1.1" 200 -
DEBUG views.py:8 我是debug级别日志
INFO views.py:9 我是infog级别日志
WARNING views.py:10 我是warning级别日志
ERROR views.py:11 我是error级别日志
CRITICAL views.py:12 我是critical级别日志
DEBUG views.py:14 falsk中记录的debug日志
INFO _internal.py:88 127.0.0.1 - - [12/Oct/2018 19:54:44] "GET / HTTP/1.1" 200 -
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 284-814-612
DEBUG views.py:11 我是debug级别日志
INFO views.py:12 我是infog级别日志
WARNING views.py:13 我是warning级别日志
ERROR views.py:14 我是error级别日志
CRITICAL views.py:15 我是critical级别日志
DEBUG views.py:17 falsk中记录的debug日志
INFO _internal.py:88 127.0.0.1 - - [12/Oct/2018 22:05:40] "GET / HTTP/1.1" 200 -
INFO _internal.py:88  * Detected change in '/home/python/Desktop/GitHub_Information/info/__init__.py', reloading
INFO _internal.py:88  * Restarting with stat
WARNING _internal.py:88  * Debugger is active!
INFO _internal.py:88  * Debugger PIN: 284-814-612
INFO _internal.py:88  * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
INFO _internal.py:88  * Restarting with stat
